cl/expr_test.go

```diff
@@ -833,6 +833,20 @@ func TestPkgMethod(t *testing.T) {
 	`, "int\n")
 }
 
+func TestPkgMethodBadCall(t *testing.T) {
+	cltest.Expect(t, `
+	import "bytes"
+	buf := bytes.NewBuffer([]byte("hello"))
+	println((&buf).String())
+	`, "", nil)
+	cltest.Expect(t, `
+	import "reflect"
+	v := reflect.ValueOf(100)
+	p := &v
+	println((&p).Kind())
```

Comment:
For the Go compiler, this example will be reported the following error:

```
calling method Kind with receiver &p (type **reflect.Value) requires explicit dereference.
```

Reply:
fixed, clean ctx.code  ast.ParenExpr  (&p) -> &p